blog/2019-03-13-KIEServerOptaPlannerTaskAssignment.adoc

```diff
@@ -0,0 +1,228 @@
+= KIE Server OptaPlanner Task Assignment
+:page-interpolate: true
+:awestruct-author: MusaTalluzi
+:awestruct-layout: blogPostBase
+:awestruct-tags: [useCase, task assigning, kie-server]
+
+https://docs.optaplanner.org/latestFinal/optaplanner-wb-es-docs/html_single/#_ch.kie.server[KIE Server]
+is a standalone server component that can be used to instantiate and execute rules and processes.
+In this blog, you will learn how to build an OptaPlanner service that implements continuous and real-time planning to solve the
+Task Assignment problem and deploy it to the KIE Server.
+
+
+== https://github.com/kiegroup/kie-server-optaplanner-task-assignment[Task Assignment KJAR]
```

Comment:
Having the subtitles be links too, is a bit unnatural, as the level 2 headers get a different color and look off:
  https://www.optaplanner.org/blog/2019/03/13/KIEServerOptaPlannerTaskAssignment.html

I'd put those link somewhere in the paragraph below it.